### ISSUE TITLE
List of all content modules

### ISSUE THREAD

**Issue by jameslibscie** [hypergraph-xyz/desktop#18]
**User story**
As a Hypergraph user
I want to see a list of all my content
so that I can get an overview of my published and unpublished work

**Workflow**
- [ ] Design
- [ ] Clickthrough
- [ ] Code
- [ ] Test

**Acceptance criteria**
- The page is titled `Workspace`
- The list displays the title, description (if any), author(s), content type (`subtype`), date modified (if own content) of each module
- If (the latest version of) a module is published, the authors' names link to their respective profile pages
- Only the latest version of each content module is displayed
- [DESIGN ONLY] The list is sorted by default by date modified (new-old)
- [DESIGN ONLY] Other sorting options that should at least be available: date created (new-old, old-new), title a-z
- This list is reachable via the `Workspace` button on the menu and is opened by default when opening the application (#15) or when closing the welcome modals (#12)
- The page includes a button for adding new content (#16)
- The list contains an icon that indicates whether the content is published or not
- [DESIGN ONLY] The list contains a bookmark icon
- If there is no content yet, display the text "Nothing here yet! Click the 'Add content' button above to get started ☝️ " If there is content, display the text "You've reached the end! ✌"

**Comment by jameslibscie** [comment]
Based on first internal clickthroughs, I have some small (suggestions for) adjustments to the designs. Normally I wouldn't like to change the specs once coding has started (unless absolutely necessary), but we want to be able to iterate on the clickthroughs.

- Rename to "Workspace" as we think it will make it clearer what you can find there

**Comment by juliangruber** [comment]
> * If there is no content yet, display the text "Nothing here yet! Click the 'Add content' button above to get started ☝️ "

What should the footer say if there _are_ modules?

**Comment by juliangruber** [comment]
> * date modified (if own content)

we don't have this in the SDK or spec yet

**Comment by juliangruber** [comment]
> * The list is sorted by default by date modified (new-old)

This we also don't have yet

**Comment by juliangruber** [comment]
> * Other sorting options that should at least be available: date created (new-old, old-new), title a-z

There's no designs for this yet..

**Comment by juliangruber** [comment]
> * The page includes a button for adding new content (#16)

This isn't a button-in-grid design available for me to reference

**Comment by juliangruber** [comment]
Everything that is available for me to be implemented has been :) ✌️

**Comment by jameslibscie** [comment]
@juliangruber Great! I've scoped out the sorting for now. Will create issues / roadmap entries for this in future milestones.

A couple of things I've noticed:
- [ ] The indicators for whether a module is published seem switched. It shows green for not published, orange for published
- [ ] Clicking the author name on a published module takes me to the workspace, rather than the profile

Looking great otherwise! :D
